frontend/lib/pages/onboarding-step-1.tsx

```diff
@@ -162,6 +166,10 @@ class OnboardingStep1WithoutContexts extends React.Component<OnboardingStep1Prop
   @autobind
   renderForm(ctx: FormContext<OnboardingStep1Input>): JSX.Element {
     const { routes } = this.props;
+    const ab: AddressAndBoroughFieldProps = {
+      addressProps: ctx.fieldPropsFor('address'),
+      boroughProps: ctx.fieldPropsFor('borough')
+    };
```

Comment:
This whole thing isn't awesome but I think it's really a code smell: what we ultimately need is just a separate `<AddressAndBorough>` component that takes this structure as props, and has the progressive enhancement built-in, rather than baking it all into the `<OnboardingStep1>` component.

However, for now I just want to get this PR finished so I'm not going to tackle that yet.

Reply:
Actually, I ended up factoring out an `AddressAndBoroughField` component in d45478b.